src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -65,23 +71,23 @@ const AuthScreen = () => {
                 } catch (err) {
                   setLoggedInUserData(null);
                 }
-              } else if(url.indexOf("?") > 0) {
-                let uri = url.substring(0, url.indexOf("?"))
-                setAdressbarURL(uri)
+              } else if (url.indexOf('?') > 0) {
```

Comment:
why these unnecessary changes are there?

Reply:
Oh, I didn't make them. Wondering why this happened

Reply:
check prettier settings !